spec/models/user_spec.rb

```diff
@@ -4,11 +4,49 @@
 
 RSpec.describe User, type: :model do
   describe 'associations' do
-    it { expect(subject).to have_many(:subscriptions) }
+    it { is_expected.to have_many(:subscriptions) }
   end
 
   describe 'validations' do
-    it { expect(subject).to validate_presence_of(:first_name) }
-    it { expect(subject).to validate_presence_of(:last_name) }
+    it { is_expected.to validate_presence_of(:first_name) }
+    it { is_expected.to validate_presence_of(:last_name) }
+  end
+
+  describe 'subscribed?' do
+    let!(:user) do
+      User.create!(
+        first_name: 'john',
+        last_name: 'doe',
+        email: 'foo@bar.com',
+        password: '123456'
+      )
+    end
+
+    context 'with an active subscription' do
+      before do
+        Subscription.create!(
+          user: user,
+          state: 'active',
+          recurrence: 'monthly'
+        )
+      end
+
+      it 'returns true' do
+        expect(user.subscribed?).to be(true)
```

Comment:
The same here about setting the subject. I always prefer having a subject set in the beginning of the description. Unless rubocop-rspec says otherwise, let's always use subject ;)

(From now on I'm not commenting on similar cases for this PR)